arangod/Network/ConnectionPool.cpp

```diff
@@ -231,15 +253,22 @@ ConnectionPtr ConnectionPool::selectConnection(std::string const& endpoint,
         // next check against the number of requests in flight
         if (c->fuerte->requestsLeft() <= limit) {
           c->lastLeased = std::chrono::steady_clock::now();
+          _successSelect++;
+          _leaseHistMSec.count(
+            duration<float, std::micro>(high_resolution_clock::now() - start).count());
```

Comment:
If we moved the durations to a `steady_clock`, we could avoid one call to `now()` here, and just reuse the result.

Reply:
Indeed. Will do.